webpack/assets/javascripts/react_app/components/CPUCoresInput/CPUCoresInput.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { WarningTriangleIcon } from '@patternfly/react-icons';
+import { HelpBlock, FormGroup, Grid, Col, Row } from 'patternfly-react';
+import { translate as __ } from '../../common/I18n';
+import NumericInput from '../common/forms/NumericInput';
+import './cpuCoresInput.scss';
+import { noop } from '../../common/helpers';
+
+const CPUCoresInput = ({
+  label,
+  recommendedMaxValue,
+  maxValue,
+  defaultValue,
+  onChange,
+  minValue,
+}) => {
+  const [value, setValue] = useState(defaultValue);
+
+  const getValidationState = useCallback(() => {
+    if (value > recommendedMaxValue) return 'warning';
+    else if (value > maxValue) return 'error';
+    return null;
+  }, [recommendedMaxValue, maxValue, value]);
+
+  const handleChange = v => {
+    setValue(v);
+    onChange();
+  };
+
+  const validationState = getValidationState();
+
+  return (
+    <FormGroup validationState={validationState}>
+      <Grid>
+        <Row>
+          <Col>
+            <NumericInput
+              value={value}
+              minValue={minValue}
+              onChange={v => handleChange(v)}
+              label={label}
+            />
+          </Col>
+        </Row>
+        {validationState != null && (
+          <HelpBlock>
+            <WarningTriangleIcon className="warning-icon" />
+            {__('Specified value is higher than recommended maximum')}
+          </HelpBlock>
+        )}
+      </Grid>
+    </FormGroup>
+  );
+};
+
+CPUCoresInput.propTypes = {
+  /** Set the label of the numeric input */
+  label: NumericInput.propTypes.label,
+  /** Set the default value of the numeric input */
+  defaultValue: PropTypes.number,
+  /** Set the recommended max value of the numeric input */
+  recommendedMaxValue: PropTypes.number,
+  /** Set the max value of the numeric input */
+  maxValue: PropTypes.number,
+  /** Set the min value of the numeric input */
+  minValue: PropTypes.number,
+  /** Set the onChange function of the numeric input */
+  onChange: PropTypes.func,
+};
+
+CPUCoresInput.defaultProps = {
+  label: '',
```

Comment:
It's a CPU core input component, shouldn't the default `label` suppose to be `CPUs`?